includes/blocks/class-sensei-blocks-initializer.php

```diff
@@ -53,8 +53,14 @@ public function maybe_initialize_blocks() {
 		if ( is_admin() ) {
 			$screen = get_current_screen();
 
+			if ( empty( $screen ) ) {
+				return;
+			}
+
+			$is_editor = $screen->is_block_editor || in_array( $screen->id, [ 'widgets', 'site-editor' ], true );
```

Comment:
This is the screen needed for Gutenberg plugin support:

```suggestion
			$is_editor = $screen->is_block_editor || in_array( $screen->id, [ 'widgets', 'site-editor', 'appearance_page_gutenberg-edit-site' ], true );
```